inc/collect.class.php

```diff
@@ -164,7 +164,7 @@ static function getSearchOptionsToAdd($itemtype = NULL) {
       $i = 5200;
 
       $pfCollect = new PluginFusioninventoryCollect();
-      foreach ($pfCollect->find(getEntitiesRestrictRequest("", $pfCollect->getTable()), 'id ASC') as $collect) {
+      foreach ($pfCollect->find(getEntitiesRestrictRequest("", $pfCollect->getTable(), '', '', true), 'id ASC') as $collect) {
```

Comment:
Sounds good to me. BTW, you should not call `find` inside the loop ;)

Reply:
I know, but here I only fix.
FOr the find, I think will do a PR only for all find in loop (I think there are many)